test/app_test.js

```diff
@@ -0,0 +1,14 @@
+const assert = require('assert');
+const request = require('supertest');
+const app = require('../app');
+
+describe('The express app', () => {
+	it('handles a GET request to /api', (done) => {
+		request(app)
+			.get('/api')
+			.end((err, res) => {
+				assert(res.body.hello === 'world');
+				done();
+			});
+	});
+});
```

Comment:
Was so much indentation intended here? Looks weird to me, but I am not necessarily opposed.